include/ExtrapolatedSmoother/ExtrapolatedSmootherTake/extrapolatedSmootherTake.h

```diff
@@ -17,7 +17,9 @@ class ExtrapolatedSmootherTake : public ExtrapolatedSmoother
 
     ~ExtrapolatedSmootherTake() override;
 
-    void extrapolatedSmoothing(Vector<double>& x, const Vector<double>& rhs, Vector<double>& temp) override;
+    void extrapolatedSmoothing(Vector<double> x,
+                               const Vector<double> rhs,
+                               Vector<double> temp) override;
```

Comment:
I think `const Vector<double>` should be `Vector<const double>` if the intention is to prevent the elements of the vector from being modified.